lib/workers/repository/dependency-dashboard.ts

```diff
@@ -453,3 +458,94 @@ function getFooter(config: RenovateConfig): string {
 
   return footer;
 }
+
+export async function getDashboardMarkdownVulnerabilities(
+  config: RenovateConfig,
+  packageFiles: Record<string, PackageFile[]>
+): Promise<string> {
+  let result = '';
+
+  if (
+    is.nullOrUndefined(config.dependencyDashboardOSVVulnerabilitySummary) ||
+    config.dependencyDashboardOSVVulnerabilitySummary === 'none'
+  ) {
+    return result;
+  }
+
+  result += '## Vulnerabilities\n\n';
+
+  const vulnerabilityFetcher = await Vulnerabilities.create();
+  const vulnerabilities = await vulnerabilityFetcher.fetchVulnerabilities(
+    config,
+    packageFiles
+  );
+
+  if (vulnerabilities.length === 0) {
+    result += 'None detected.\n\n';
+    return result;
+  }
+
+  const unresolvedVulnerabilities = vulnerabilities.filter((value) =>
+    is.nullOrUndefined(value.fixedVersion)
+  );
+
+  result += `${unresolvedVulnerabilities.length} of a total of ${vulnerabilities.length} CVEs have no fixes in this repository.\n`;
```

Comment:
```suggestion
  result += `${unresolvedVulnerabilities.length} out of ${vulnerabilities.length} CVEs need to be fixed by you, because there is no updated dependency that fixes the CVE.\n`;
```

We can simplify a bit with `x out of y`. We should tell our users what they need to do if Renovate cant create a PR which updates to a good version of the vulnerable dependency.